cloudimg/aws.py

```diff
@@ -658,22 +658,28 @@ def import_snapshot(self, obj, snapshot_name, tags=None):
 
         return snapshot
 
-    def copy_ami(self, image_id, image_name, image_region,):
+    def copy_ami(self, image_id, image_name, image_region, tags):
         """
         Create copy of an AMI.
 
         Args:
             image_id (str): AMI Id of the image to copy
             name (str): Name of new image
             region(str): Region for new Image.
+            tags(list, optional): Tags to be associated with new ami.
```

Comment:
It's documented as optional, but it's not actually optional in the python sense - `copy_ami` now *requires* a value for `tags`, and 100% of existing callers won't be providing that, so they'll just crash.  This is a severe backwards-incompatible change.

It needs to be made genuinely optional.

Reply:
This comment still seems to be valid: if someone calls this method without providing tags, it'll crash, so it's still not backwards-compatible.

What I'm saying is that the signature should have:

```
tags=None
```